scripts/realsense-camera-formats-bionic-hwe-5.4.patch

```diff
@@ -132,14 +141,15 @@ index 58868d7129eb..c29033b8439c 100644
 +	case V4L2_PIX_FMT_INZC:		descr = "Planar Depth/Confidence (INZC)"; break;
 +	case V4L2_PIX_FMT_PAIR:		descr = "Relative IR (PAIR)"; break;
 +	case V4L2_PIX_FMT_Z16H:		descr = "Z16 Huffman Compression"; break;
++	case V4L2_PIX_FMT_Y16I:		descr = "Interleaved 16-bit Greyscale"; break;
```

Comment:
Would it make more sense to inject the line immedeately after Y12I ?

Reply:
done